contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1242,11 +1260,25 @@ code example 2 <2>
 <2> A note about the second example value.
 ....
 
-* If you must provide additional information on what a line of a code block
-represents and the use of callouts is impractical, you can use a description list
-to provide information about the variables in the code block. Using callouts
-might be impractical if a code block contains too many conditional statements to
-easily use numbered callouts or if the same note applies to multiple lines of the codeblock.
+* If your command contains multiple lines and uses callout annotations, you must comment out the callout(s) in the codeblock, as shown in the following example:
++
+....
+To scale based on the percent of CPU utilization, create a `HorizontalPodAutoscaler` object for an existing object:
+
+[source,terminal]
+----
+$ oc autoscale <object_type>/<name> \// <1>
+  --min <number> \// <2>
+  --max <number> \// <3>
+  --cpu-percent=<percent> <4>
```

Comment:
These show up in the output as follows:
```
$ oc autoscale <object_type>/<name> \// (1)
  --min <number> \// (2)
  --max <number> \// (3)
  --cpu-percent=<percent> (4)
```

The ones on lines 1256-1257 are angle brackets, I don't understand why it shows these as parens :thinking: